### PR TITLE
feat(navbar): update current navbar with new component oui-navbar

### PR DESCRIPTION
**Pull request by AxelPeter**
UU-589

### Before reviewing, need this PR to be merged first !
https://github.com/ovh-ux/ovh-manager-web/pull/267

### CODE REVIEW

**Comment by AxelPeter** [comment]
Temporary, i change the target branch to `chore/update-oui-kit` to avoid too many changed files.
I'll put back to `develop` when this PR is merged: https://github.com/ovh-ux/ovh-manager-web/pull/267

File: client/app/components/sidebar-menu/sidebar-menu.less [hunk 1]
```diff
@@ -10,8 +10,15 @@ button, toggle-class {
 }
 
 #sidebar-menu {
+    @sidebar-width: 300px;
+
+    width: @sidebar-width;
     z-index: 1020;
```
Comment (FredericEspiau) [inline client/app/components/sidebar-menu/sidebar-menu.less]: Still useful ?

File: client/app/components/user/session.controller.js [hunk 2]
```diff
@@ -1,45 +1,32 @@
...
         // FIX for /me/alerts
         $scope.$on("Navigator.navigationInformationsChange", (e, data) => {
             $scope.isLeftMenuVisible = data && data.leftMenuVisible;
         });
 
-        translator.setTitle($scope.tr("global_app_title"));
+        // Scroll to anchor id
+        $scope.scrollTo = (id) => {
+            // Set focus to target
+            $document.find(`#${id}`)[0].focus();
```
Comment (FredericEspiau) [inline client/app/components/user/session.controller.js]: Maybe you should check if the `id` exists before using the element returned by `find` but I don't know that happens when the `id` isn't found

File: client/app/components/user/session.service.js [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+    getLanguageChangingMenuObject () {
+        const languageChangingMenuObject = _(this.LANGUAGES)
+            .filter((language) => _(language).has("name", "value"))
+            .map((language) => ({
+                title: language.name,
+                click: (callback) => {
+                    localStorage["univers-selected-language"] = language.value;
+                    window.location.reload();
+
+                    if (typeof callback === "function") {
```
Comment (FredericEspiau) [inline client/app/components/user/session.service.js]: `https://lodash.com/docs/3.10.1#isFunction`

File: client/app/components/sidebar-menu/sidebar-menu.less (hunk 1 above)
Comment (AxelPeter) [reply client/app/components/sidebar-menu/sidebar-menu.less]: About what `@sidebar-width` ?
It's used a bit further in the file.

File: client/app/components/user/session.service.js (hunk 3 above)
Comment (AxelPeter) [reply client/app/components/user/session.service.js]: Yes, done

File: client/app/components/user/session.controller.js (hunk 2 above)
Comment (AxelPeter) [reply client/app/components/user/session.controller.js]: It's only used once for the moment. But yes, we could. 
Updated

File: client/app/index.ejs [hunk 4]
```diff
@@ -54,16 +54,34 @@
...
+                data-aside-links="::navbar.internalLinks">
+            </oui-navbar>
+
+            <div data-sidebar-menu></div>
+
+            <div class="main-wrapper pb-5">
+
+                <!-- Skip content target -->
+                <div class="skiptarget">
+                    <a id="maincontent" href="">-</a>
```
Comment (jleveugle) [inline client/app/index.ejs]: Anchor ?

Comment (AxelPeter) [reply client/app/index.ejs]: Yes, to skip content.
I took this way of doing from here: https://www.w3.org/TR/WCAG20-TECHS/G1.html

File: client/app/components/user/session.service.js [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+                    productId: elt.name
+                }
+            })
+        );
+
+        // Office products
+        const officeLinks = [];
+        const officeProducts = _.sortBy(products.licenseOffice, (elt) => angular.lowercase(elt.displayName || elt.name));
+
+        _.forEach(officeProducts, (elt) =>
```
[the remark this answers is not in the document]
Comment (AxelPeter) [reply client/app/components/user/session.service.js]: Done :)

File: client/app/components/user/session.controller.js [hunk 6]
```diff
@@ -1,45 +1,34 @@
 angular.module("App").controller("SessionCtrl", [
```
[the remark this answers is not in the document]
Comment (AxelPeter) [reply client/app/components/user/session.controller.js]: I thought it was only necessary for function. Must be injected with grunt.
Done :)

File: client/app/css/less/common/navbar.less [hunk 7]
```diff
@@ -0,0 +1,93 @@
+.oui-navbar {
+    // Logo
+    .navbar-logo {
+        height: 3rem;
+        margin-top: -.3rem; // For a better centering
```
[the remark this answers is not in the document]
Comment (AxelPeter) [reply client/app/css/less/common/navbar.less]: Done with flex

**Comment by AxelPeter** [comment]
https://github.com/ovh-ux/ovh-manager-web/pull/267 has been merged. I change back the target branch to `develop`

**Pull request merged**